rosdep/python.yaml

```diff
@@ -1573,6 +1573,16 @@ python-elasticsearch:
   opensuse: [python-elasticsearch]
   ubuntu:
     bionic: [python-elasticsearch]
+python-emoji-pip:
```

Comment:
do you want to make it available for python3?
In that case, the key should be `python3-emoji-pip`.

Reply:
There's an ubuntu debian package available: https://packages.ubuntu.com/jammy/python3-emoji.

So consider adding a `python3-emoji` key instead